cmd/snap-bootstrap/initramfs_systemd_mount_test.go

```diff
@@ -277,16 +473,22 @@ func (s *doSystemdMountSuite) TestDoSystemdMount(c *C) {
 				case arg == "--property=Before=initrd-fs.target":
 					foundBeforeInitrdfsTarget = true
 				case strings.HasPrefix(arg, "--options="):
-					for _, opt := range strings.Split(strings.TrimPrefix(arg, "--options="), ",") {
-						switch opt {
-						case "nosuid":
+					for _, opt := range osutil.SplitSystemdMountOptions(strings.TrimPrefix(arg, "--options=")) {
```

Comment:
If we forbid `,` then we do not need to worry about the split. I would introduce this function only in follow up PRs if we decide to allow some of the escaping.

Reply:
I removed it.